docs/MapColonies/Raster/services/polygon_parts.md

```diff
@@ -0,0 +1,877 @@
+          "maxOccurs": 1,
+          "minOccurs": 0,
+          "nillable": true,
+          "type": "xsd:string",
+          "localType": "string"
+        },
+        {
+          "name": "description",
+          "maxOccurs": 1,
+          "minOccurs": 0,
+          "nillable": true,
+          "type": "xsd:string",
+          "localType": "string"
+        },
+        {
+          "name": "geometry",
+          "maxOccurs": 1,
+          "minOccurs": 0,
+          "nillable": true,
+          "type": "gml:Polygon",
+          "localType": "Polygon"
+        }
+      ]
+    }
+  ]
+}
+```
+
+</details>
+
+The response is the XSD (in xml response) describing the `orthophoto_mosaic_base_orthophoto_best_polygon_parts` `FeatureType`.
+
+This is a description of a orthophoto_mosaic_base_orthophoto_best_polygon_parts feature that include all polygon parts of the `ORTHOPHOTO_MOSAIC_BASE-OrthophotoBest` catalog layer, it contains these properties, each property has its own type, nillable flag value and min\max occurs - these describing the requireness of the property and possible repeatness.
+
+Now that we hold the structure of the `orthophoto_mosaic_base_orthophoto_best_polygon_parts` FeatureType we're able to query layer's polygon parts features by a set of parameters using the `GetFeature` operation, let's see some examples:
+
+1. let's retrieve 2 of polygon parts for provided featureType `orthophoto_mosaic_base_orthophoto_best_polygon_parts`, that include actually total of 4 features:
+
+```
+<POLYGON_PARTS_QUERY_SERVICE_URL>/wfs?
+    service=wfs&
+    version={WFS_SERVICE_VERSION}&
+    request=GetFeature&
+    count=2&
+    typeName=orthophoto_mosaic_base_orthophoto_best_polygon_parts&
+    outputFormat=application/json
+```
+
+<details>
+  <summary>Response</summary>
+
+```json
+{
+    "type": "FeatureCollection",
+    "features": [
+        {
+            "type": "Feature",
+            "id": "orthophoto_mosaic_base_orthophoto_best_polygon_parts.4",
+            "geometry": {
+                "type": "Polygon",
+                "coordinates": [
+                    [
+                        [
+                            25,
+                            90
+                        ],
+                        [
+                            25,
+                            -90
+                        ],
+                        [
+                            -180,
+                            -90
+                        ],
+                        [
+                            -180,
+                            90
+                        ],
+                        [
+                            25,
+                            90
+                        ]
+                    ]
+                ]
+            },
+            "geometry_name": "geometry",
+            "properties": {
+                "partId": 1,
+                "recordId": "cc1ccbc6-cb96-4006-82b3-6a74fc2e7abd",
+                "productId": null,
```

Comment:
"productId" and "producType" can't be null... (fix in all responses)

Reply:
done